_docs_v5/4_guides/scripting.md

```diff
@@ -2883,27 +2883,6 @@ set_stdout_max_lines(2000)
 
 These methods allow the user to debug scripts with ScriptRunner.
 
-### insert_return
-
-Inserts a ruby return statement into the currently executing context. This can be used to break out of methods early from the ScriptRunner Debug prompt.
-
-Syntax:
-
-```ruby
-insert_return (<return value (optional)>, ...)
-```
-
-| Parameter    | Description                                          |
-| ------------ | ---------------------------------------------------- |
-| return value | One or more values that are returned from the method |
-
-Example:
-
-```ruby
-insert_return()
-insert_return(5, 10)
-```
-
```

Comment:
We don't have insert_return implemented. Should we bring it back?

Reply:
No this got removed.